helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -35,9 +35,11 @@ receivers:
     {{- if eq (include "splunk-otel-collector.distribution" .) "openshift" }}
     distribution: openshift
     {{- end }}
-  {{- if and $clusterReceiver.eventsEnabled (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
-  k8s_events:
+  {{- if and $clusterReceiver.objectsEnabled (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
+  k8sobjects:
     auth_type: serviceAccount
+    objects:
+{{ include "splunk-otel-collector.objectsEnabled" . | nindent 6 }}
```

Comment:
Shouldn't it be `splunk-otel-collector.clusterRole.clusterReceiver.objects`?

Reply:
Right! actually, I'd even remove `clusterRole` element. To make it `splunk-otel-collector.clusterReceiver.objects`